src/main/java/com/jay/shoppingmall/service/SellerService.java

```diff
@@ -345,8 +382,10 @@ public void sellerDefaultSettingSave(SellerDefaultSettingsRequest request, User
                 .orElseThrow(() -> new SellerNotFoundException("판매자가 아닙니다"));
 
         if (sellerRepository.existsByCompanyName(request.getCompanyName())) {
-            if (!seller.getCompanyName().equals(request.getCompanyName()))
+            if (!seller.getCompanyName().equals(request.getCompanyName())) {
+                log.info("Seller tried to save or change its company name but it was duplicated. sellerCompanyName = '{}', requestCompanyName = '{}'", seller.getCompanyName(), request.getCompanyName());
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This code contains a potential information leakage in the handling of the following *logged data*: (Seller seller) -&gt; (Company Name),(SellerDefaultSettingsRequest request) -&gt; (Company Name). 

**Fix**
Release data more selectively, or declassify the data before it&#39;s released. 

**More info**
[Learn more about information leakage](https://cwe.mitre.org/data/definitions/532.html) (external link).